pkg/minikube/bootstrapper/bootstrapper.go

```diff
@@ -32,7 +31,8 @@ type Bootstrapper interface {
 	UpdateCluster(config.KubernetesConfig) error
 	RestartCluster(config.KubernetesConfig) error
 	DeleteCluster(config.KubernetesConfig) error
-	GetClusterLogsTo(follow bool, out io.Writer) error
+	// LogCommands returns a map of log type to a command which will display that log.
+	LogCommands(int, bool) map[string]string
```

Comment:
I would consider wrapping the int and bool into a `LogOpts` struct instead - that way you can introduce new fields without mandatorily having to change it in all implementations and it could also help readability a bit more here.

Reply:
Excellent idea! Done.